release_dashboard/test/widgets/common/dialog_prompt_test.dart

```diff
@@ -170,9 +303,95 @@ void main() {
 
       await tester.tap(find.byType(ElevatedButton));
       await tester.pumpAndSettle();
-      await tester.tap(find.text(rightOption));
+      await tester.tap(find.text(leftButtonTitle));
+      await tester.pumpAndSettle();
+      expect(isLeftButtonCallbackCalled, equals(true));
+      expect(isRightButtonCallbackCalled, equals(false));
+    });
+
+    testWidgets('Right button enables when the user input is correct, disables otherwise', (WidgetTester tester) async {
```

Comment:
Another test is if the user enters the input confirmation string, but has extra pieces (like `clean release please`), that should be denied

Reply:
It is checked here: https://github.com/Yugue/cocoon/blob/e505784161eba78fcd81519ad625f68768922496/release_dashboard/test/widgets/common/dialog_prompt_test.dart#L343

After the user has entered `clean release`, the button enables. Then if the user enters an extra `@`, the button becomes disabled.

Reply:
This is an example where smaller tests would help (or adding comments to what the behavior is being tested)

Reply:
Added, please take another look.